.github/workflows/build.yml

```diff
@@ -39,11 +39,21 @@ jobs:
       matrix:
         java: [ 21, 17, 11, 8 ]
     name: Compile and Check Java ${{ matrix.java }}
+    outputs:
+      is-draft: ${{ steps.check-draft-pr.outputs.is-draft }}
     steps:
       - name: Env
         run: printenv
         env:
           GITHUB_CONTEXT: ${{ toJson(github) }}
+      - name: Check for Draft PR
+        id: check-draft-pr
+        if: |
+          github.event_name == 'pull_request' && (
+            github.event.pull_request.draft ||
+            contains(github.event.pull_request.title, 'WIP')
```

Comment:
Yea, that's a good point. I went with the title since that's something non-committers can update (unlike labels). I'm fine leaving this out for now and just using the Draft status.